SessionMessagingKit/Database/Storage+Messaging.swift

```diff
@@ -42,6 +42,7 @@ extension Storage {
             }
             tsMessage = tsOutgoingMessage
         } else {
+            if let _ = TSIncomingMessage.find(withAuthorId: message.sender!, timestamp: message.sentTimestamp!, transaction: transaction) { return nil }
```

Comment:
Please change this to be `if TSIncomingMessage.find(withAuthorId: message.sender!, timestamp: message.sentTimestamp!, transaction: transaction) != nil {` rather than `if let _ = ...` (it's generally the more accepted convention)

Reply:
Done. :)